src/components/results/sensitivity-analysis/sensitivity-analysis-result-tab.js

```diff
@@ -69,23 +77,39 @@ const SensitivityAnalysisResultTab = ({ studyUuid, nodeUuid }) => {
                 sensiKindIndex={sensiKindIndex}
                 setSensiKindIndex={handleSensiKindIndexChange}
             />
-            <Tabs value={nOrNkIndex} onChange={handleSensiNOrNkIndexChange}>
-                {SensitivityResultTabs.map((tab) => (
-                    <Tab label={tab.label} />
-                ))}
-            </Tabs>
-            <PagedSensitivityAnalysisResult
-                nOrNkIndex={nOrNkIndex}
-                sensiKindIndex={sensiKindIndex}
-                studyUuid={studyUuid}
-                nodeUuid={nodeUuid}
-                updateFilter={updateFilter}
-                filterSelector={filterSelector}
-                onSortChanged={onSortChanged}
-                sortConfig={sortConfig}
-                page={page}
-                setPage={setPage}
-            />
+            {sensiKindIndex < 3 && (
+                <>
+                    <Tabs
+                        value={nOrNkIndex}
+                        onChange={handleSensiNOrNkIndexChange}
+                    >
+                        {SensitivityResultTabs.map((tab) => (
+                            <Tab label={tab.label} />
+                        ))}
+                    </Tabs>
+                    <PagedSensitivityAnalysisResult
+                        nOrNkIndex={nOrNkIndex}
+                        sensiKindIndex={sensiKindIndex}
+                        studyUuid={studyUuid}
+                        nodeUuid={nodeUuid}
+                        updateFilter={updateFilter}
+                        filterSelector={filterSelector}
+                        onSortChanged={onSortChanged}
+                        sortConfig={sortConfig}
+                        page={page}
+                        setPage={setPage}
+                    />
+                </>
+            )}
+            {sensiKindIndex === 3 &&
```

Comment:
This whole "sensiKindIndex" doesn't work to track the tabIndex.
Maybe we should rename that to "tabIndex", and use the FUNCTION_TYPES instead of sensiKindIndex in the PagedSensitivityAnalysisResult component ?

Reply:
sensiKindIndex is used as a tabIndex and works.
I prefer not to change existing code in this PR, at this time.